webapp/src/pages/LoginPage/LoginPage.tsx

```diff
@@ -14,9 +14,13 @@ import {
 import { ResponseError } from '../../shared/generated';
 import { authApi } from '../../shared/services/ApiService';
 import { LOGIN_EP_URL, REGISTER_URL, USERS_URL } from '../../shared/urls';
-import { SubmitStatus } from '../../shared/types';
 import './LoginPage.css';
 
+type SubmitStatus = {
```

Comment:
every MR is changing this file, so you'll probably have conflicts. But you'll figure it out, I'm sure.

Reply:
Yes, and sorry, It was a little mess but I will be careful when merging